cmd/jimmsrv/service/service.go

```diff
@@ -408,7 +408,7 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 
 	s.mux.Mount("/metrics", promhttp.Handler())
 
-	s.mux.Mount("/rebac", middleware.AuthenticateRebac(rebacBackend.Handler(""), &s.jimm))
+	s.mux.Mount("/rebac", middleware.AuthenticateRebac("/rebac", rebacBackend.Handler(""), &s.jimm))
```

Comment:
Can we make this follow the pattern of the other httphandlers in jimmhttp? I asked @SimoneDutto to do the same on his PR/ Maybe we can't with this because of the lib but would be nice to try

Reply:
What do you mean? To make it independent of the base URL?